src/vaults/vaults.service.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { VaultsService } from './vaults.service';
+import { BadgerSDK } from '../sdk';
+import { VaultVersion } from './enums';
+import { Network } from '../config';
+
+describe('vaults.service', () => {
+  let sdk: BadgerSDK;
+
+  beforeAll(async () => {
+    sdk = new BadgerSDK({
+      network: Network.Ethereum,
+      // TODO: Find out how to replace RPC calls with proper mocks
+      provider: 'https://rpc.ftm.tools/',
+    });
+  });
+
+  describe('getVaultStrategy', () => {
+    it('should return v1.5 strategy', async function () {
+      const service = new VaultsService(sdk);
```

Comment:
You can have 
‘let service’ and use a ‘beforeEach’ to set it each time

Reply:
Fixed